v2/go.mod

```diff
@@ -2,4 +2,12 @@ module github.com/googleapis/gax-go/v2
 
 go 1.11
 
-require google.golang.org/grpc v1.19.0
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/protobuf v1.5.0
+	github.com/stretchr/testify v1.7.0
+	google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
+	google.golang.org/grpc v1.19.0
+	google.golang.org/protobuf v1.27.1 // indirect
```

Comment:
If we are to use a protobuf-go library, this is the one we want to use.

Docs: https://pkg.go.dev/google.golang.org/protobuf